.changeset/modern-worms-exercise.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
This should have been the changelog, you can click the comment on the PR to auto populate with the commit message

Reply:
It said to use `--empty` since there is no functional, relevant change to cadl ranch.

Reply:
It won’t bum the version when releasing without change

Reply:
make a patch change